src/loaders/trackedEntityLoader.js

```diff
@@ -10,27 +10,21 @@ import {
 import { getProgramStatuses } from '../constants/programStatuses.js'
 import { getOrgUnitsFromRows } from '../util/analytics.js'
 import { apiFetch } from '../util/api.js'
+import {
+    GEO_TYPE_POINT,
+    GEO_TYPE_POLYGON,
+    GEO_TYPE_MULTIPOLYGON,
+} from '../util/geojson.js'
 import { getDataWithRelationships } from '../util/teiRelationshipsParser.js'
 import { formatStartEndDate, getDateArray } from '../util/time.js'
 
-const fields = [
-    'trackedEntityInstance~rename(id)',
-    'featureType',
-    'coordinates',
-]
-
-// Mapping netween DHIS2 types and GeoJSON types
-const geometryTypesMap = {
-    POINT: 'Point',
-    POLYGON: 'Polygon',
-    MULTI_POLYGON: 'MultiPolygon',
-}
+const fields = ['trackedEntity~rename(id)', 'geometry']
 
 // Valid geometry types for TEIs
-const geometryTypes = Object.keys(geometryTypesMap)
+const geometryTypes = [GEO_TYPE_POINT, GEO_TYPE_POLYGON, GEO_TYPE_MULTIPOLYGON]
```

Comment:
Instead of including a comment, you could name this array `validTEIGeometryTypes`

Reply:
Addressed here: https://github.com/dhis2/maps-app/pull/3346/commits/fb59ae7f2f693b09c4312726e64b022965d8e00b